docs/user/azure/encryption.md

```diff
@@ -0,0 +1,124 @@
+export KEYVAULT_NAME="<Keyvault Name>"
+export KEYVAULT_KEY_NAME="<Keyvault Key Name>"
+export DISK_ENCRYPTION_SET_NAME="<Disk Encryption Set Name>"
+export CLUSTER_SP_ID="<Service Principal ID>"
+```
+
+### Register features
+
+Encryption at host is in preview, and must be enabled to be used.
+
+```
+az feature register --namespace "Microsoft.Compute" --name "EncryptionAtHost"
+az feature show --namespace Microsoft.Compute --name EncryptionAtHost
+az provider register -n Microsoft.Compute
+```
+
+### Create resource group
+```
+az group create --name $RESOURCEGROUP --location $LOCATION
+```
+
+### Create key vault
+```
+az keyvault create -n $KEYVAULT_NAME -g $RESOURCEGROUP -l $LOCATION \
+        --enable-purge-protection true --enable-soft-delete true
+
+az keyvault key create --vault-name $KEYVAULT_NAME -n $KEYVAULT_KEY_NAME \
+        --protection software 
+```
+
+### Create an instance of a DiskEncryptionSet
+```
+KEYVAULT_ID=$(az keyvault show --name $KEYVAULT_NAME --query "[id]" -o tsv)
+    
+KEYVAULT_KEY_URL=$(az keyvault key show --vault-name $KEYVAULT_NAME --name \
+        $KEYVAULT_KEY_NAME --query "[key.kid]" -o tsv)
+    
+az disk-encryption-set create -n $DISK_ENCRYPTION_SET_NAME -l $LOCATION -g \
+        $RESOURCEGROUP --source-vault $KEYVAULT_ID --key-url $KEYVAULT_KEY_URL
+```
+
+### Grant the DiskEncryptionSet resource access to the key vault
+```
+DES_IDENTITY=$(az disk-encryption-set show -n $DISK_ENCRYPTION_SET_NAME -g \
+        $RESOURCEGROUP --query "[identity.principalId]" -o tsv)
+
+az keyvault set-policy -n $KEYVAULT_NAME -g $RESOURCEGROUP --object-id \
+        $DES_IDENTITY --key-permissions wrapkey unwrapkey get
+```
+
+### Grant service principal reader permissions to the DiskEncryptionSet
+```
+DES_RESOURCE_ID=$(az disk-encryption-set show -n $DISK_ENCRYPTION_SET_NAME -g \
+        $RESOURCEGROUP --query "[id]" -o tsv)
+
+az role assignment create --assignee $CLUSTER_SP_ID --role Owner --scope \
+        $DES_RESOURCE_ID -o jsonc
+```
+
+### Configure the disk encryption set in your install-config.yaml
+```yaml
+apiVersion: v1
+baseDomain: my.basedomain.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform:
+    azure:
+      encryptionAtHost: true
+      osDisk:
+        diskEncryptionSet:
+          resourceGroup: diskEncryptionResourceGroup
+          name: diskEncryptionSetName
+  replicas: 3
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform:
+    azure:
+      encryptionAtHost: true
+      osDisk:
+        diskEncryptionSet:
+          resourceGroup: diskEncryptionResourceGroup
+          name: diskEncryptionSetName
+  replicas: 3
+...
+```
+
+### Run the installer
+```
+openshift-install create cluster --log-level debug
+```
+
+
+### Update the annotations on the default storage class
+```
+oc patch storageclass managed-premium -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
```

Comment:
Do we know which is the default SC now after customers disable storageclass that we create as default?

Reply:
Also in 4.11 - I do not think this is the default storageclass anymore.